docs/OpenShift GitOps CLI User Guide.md

```diff
@@ -0,0 +1,636 @@
+# OpenShift GitOps CLI User Guide
+OpenShift GitOps `argocd` CLI enables users to manage OpenShift GitOps from a terminal. Using the CLI tool, users can manage ArgoCD resources (Application, ApplicationSets, AppProjects, User accounts, GPG keys etc) from a client terminal.
+
+**Note:** Both the archives and the RPMs contain the `argocd` executable binary file.
+
+**WARNING** 	
+Red Hat OpenShift GitOps with the `argocd` CLI tool is a Technology Preview feature only.
+
+**Tip:** This guide assumes you have a grounding in the tools that OpenShift GitOps is based on. Please read [about Red Hat OpenShift GitOps](https://docs.openshift.com/gitops/1.11/understanding_openshift_gitops/about-redhat-openshift-gitops.html) to learn about these tools.
+
+## Installing OpenShift GitOps CLI (argocd)
+Use the CLI tool to manage Red Hat OpenShift GitOps from a terminal. You can install the CLI tool on different platforms.
+
+### Installing the Red Hat OpenShift GitOps CLI on Linux using an RPM
+For Red Hat Enterprise Linux (RHEL) version 8, you can install the Red Hat OpenShift GitOps CLI as an RPM.
```

Comment:
@anandf what about Red Hat Enterprise Linux (RHEL) version 9? is it not applicable?

Should we update this to RHEL 9?

Reply:
I think its better not to mention the version at all. Customers can be either in RHEL 8/9 version.

Reply:
@anandf version must be mentioned. something like, For Red Hat Enterprise Linux (RHEL) version 8 and later ?

No mentioning is not a good practice.